src/index.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import GlobalStyles from './GlobalStyles'
+import GlobalStyles from './GlobalStyles';
+
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+      <GlobalStyles />
```

Comment:
Is it necessary to put the <GlobalStyles /> inside the App? I think you can put in next to the app.